pkg/apis/servicecatalog/testing/fuzzer.go

```diff
@@ -65,6 +66,17 @@ func createParameter(c fuzz.Continue) (*runtime.RawExtension, error) {
 	return &runtime.RawExtension{Raw: b}, nil
 }
 
+func createParameters(c fuzz.Continue) *servicecatalog.Parameters {
```

Comment:
@duglin Yeah though about that as well, but then deliberately chose this name to reuse this method for bindings as well in the future (as a separate PR of course).
Still agree that probably better to call it `createInstanceParameters` for now and rename after if needed.